openpype/modules/kitsu/actions/launcher_show_in_kitsu.py

```diff
@@ -0,0 +1,130 @@
+import webbrowser
+
+from openpype.pipeline import LauncherAction
+from openpype.modules import ModulesManager
+from openpype.client import get_project, get_asset_by_name
+
+
+class ShowInKitsu(LauncherAction):
+    name = "showinkitsu"
+    label = "Show in Kitsu"
+    icon = "external-link-square"
+    color = "#e0e1e1"
+    order = 10
+
+    @staticmethod
+    def get_kitsu_module():
+        return ModulesManager().modules_by_name.get("kitsu")
+
+    def is_compatible(self, session):
+        if not session.get("AVALON_PROJECT"):
+            return False
+
+        return True
+
+    def process(self, session, **kwargs):
+
+        # Context inputs
+        project_name = session["AVALON_PROJECT"]
+        asset_name = session.get("AVALON_ASSET", None)
+        task_name = session.get("AVALON_TASK", None)
+
+        project = get_project(project_name=project_name,
+                              fields=["data.zou_id"])
+        if not project:
+            raise RuntimeError(f"Project {project_name} not found.")
+
+        project_zou_id = project["data"].get("zou_id")
+        if not project_zou_id:
+            raise RuntimeError(f"Project {project_name} has no "
+                               f"connected ftrack id.")
+
+        asset_zou_data = None
+        task_zou_id = None
+        asset_zou_name = None
+        asset_zou_id = None
+        asset_zou_type = 'Assets'
```

Comment:
This is probably OCD - but could we reorder these:
```suggestion
        asset_zou_name = None
        asset_zou_id = None
        asset_zou_type = 'Assets'
        task_zou_id = None
```

That way they go from big to small (as in, tasks are inside assets).